<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>statementItem</class>
 <widget class="QWidget" name="statementItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="startTimeLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>开始时间：11-11 22:22</string>
   </property>
  </widget>
  <widget class="QLabel" name="stopTimeLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="startTimeLabel_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>开始温度：70°C</string>
   </property>
  </widget>
  <widget class="QLabel" name="finishTemLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="windLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>风速：高</string>
   </property>
  </widget>
  <widget class="QLabel" name="eneryLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>能量：999度</string>
   </property>
  </widget>
  <widget class="QLabel" name="costLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>88</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>总费用：
999元</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
